rest/taskana-rest-spring/src/main/java/pro/taskana/monitor/rest/MonitorController.java

```diff
@@ -78,6 +81,54 @@
     return ResponseEntity.status(HttpStatus.OK).body(report);
   }
 
+  /**
+   * This endpoint generates a Workbasket Report by priority. This Report currently
+   *
+   * <p>Each Row represents a Workbasket.
+   *
+   * <p>Each Column Header represents a priority range.
+   *
+   * <table>
+   *   <caption><b>Default ranges</b></caption>
+   *  <tr>
+   *    <td> high </td> <td>priority &gt; 500</td>
+   *  </tr>
+   *  <tr>
+   *    <td> medium </td> <td> 250 &ge; priority &le; 500</td>
+   *  </tr>
+   *  <tr>
+   *    <td> low </td> <td> priority &lt; 250</td>
+   *  </tr>
+   * </table>
```

Comment:
does this look pretty? Did you check?

Reply:
![image](https://user-images.githubusercontent.com/19949441/131470606-454a93a2-fd76-4506-aedb-a9920ae4e482.png)
I think it looks fabulous